Command/CKEditorInstallerCommand.php

```diff
@@ -39,7 +39,7 @@ public function __construct(CKEditorInstaller $installer = null)
     {
         parent::__construct();
 
-        $this->installer = $installer ?: new CKEditorInstaller();
+        $this->installer = $installer ? : new CKEditorInstaller();
```

Comment:
`$this->installer = $installer ?? new CKEditorInstaller();`

Reply:
https://github.com/egeloen/IvoryCKEditorBundle/blob/master/composer.json#L14 not sure it can

Reply:
Oh sorry, I thought support below 7.0 was dropped.